index.md

```diff
@@ -12,7 +12,7 @@ footer: >
 {% include box.html type="start" h="2" title="Summary" class="full" %}
 {:/}
 
-This page describes steps to help you report website accessibility problems:
+This document describes steps to help you report website accessibility problems:
```

Comment:
You use "document" later on. Either change this to "document" or the other to "page", to be consistent.